hostapd/hostapd_cli_cmds.c

```diff
@@ -173,9 +173,9 @@ static int hostapd_cli_cmd_sta(struct wpa_ctrl *ctrl, int argc, char *argv[])
 		return -1;
 	}
 	if (argc > 1)
-		snprintf(buf, sizeof(buf), "STA %s %s", argv[0], argv[1]);
+		os_snprintf(buf, sizeof(buf), "STA %s %s", argv[0], argv[1]);
```

Comment:
can you please move this rename to a separate commit, it's not matching the commit description.

Reply:
I move this rename to the origin commit(2nd commit), which added hostapd_cli_cmds.c as new file. Is it ok for you?